frontend/css/entity-pages.less

```
@@ -28,7 +28,11 @@
       display: flex;
       flex-direction: column;
       min-width: 200px;
-      max-height: 410px;
+      max-height: 29em;
```

Handles cases for tablet size screens.
```suggestion
      min-width: 15em;
      max-height: 29em;
```